sdks/python/apache_beam/yaml/examples/README.md

```diff
@@ -23,36 +23,73 @@
 * [Examples Catalog](#examples-catalog)
   * [Wordcount](#wordcount)
   * [Transforms](#transforms)
-    * [Element-wise](#element-wise)
     * [Aggregation](#aggregation)
+    * [Blueprints](#blueprints)
+    * [Element-wise](#element-wise)
+    * [IO](#io)
+    * [ML](#ml)
+
 <!-- TOC -->
 
+## Prerequistes
+Build this jar for running with the run command in the next stage:
+```
+cd <path_to_beam_repo>/beam; ./gradlew sdks:java:io:google-cloud-platform:expansion-service:shadowJar
+```
+
+## Example Run
 This module contains a series of Beam YAML code samples that can be run using
 the command:
 ```
 python -m apache_beam.yaml.main --pipeline_spec_file=/path/to/example.yaml
 ```
 
+Depending on the yaml pipeline, the output maybe sent to the command line or a
```

Comment:
> the output maybe sent to the command line

do you mean something like 'the output may be emitted to standard output'

Reply:
updated, thanks